language/evm/move-to-yul/src/functions.rs

```diff
@@ -43,11 +43,12 @@ impl<'a> FunctionGenerator<'a> {
         let fun = &ctx.env.get_function(fun_id.to_qualified_id());
         // TODO: change back to is_native_or_intrinsic if we decide to implement
         // intrinsic functions (such as reverse, contains, is_empty) in the Vector module as well
-        if fun.is_native() {
+        if fun.is_native() || fun.get_full_name_str().contains("send_") {
```

Comment:
It would be nice to create a compilation flag to gate this feature IMO so that this if would only be active when a rust feature flag is set, similar to how we gate the table extension

Reply:
I added a gate when _function_ is called so that only when #[actor] is specified, the feature is activated.